pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook.go

```diff
@@ -153,6 +151,15 @@ func (v *fleetResourceValidator) handleNamespace(req admission.Request) admissio
 	return admission.Allowed("namespace name doesn't begin with fleet/kube prefix so we allow all operations on these namespaces")
 }
 
+// handleEvent allows/denies request to modify event after validation.
+func (v *fleetResourceValidator) handleEvent(ctx context.Context, req admission.Request) admission.Response {
+	if isFleetMemberNamespace(req.Namespace) {
```

Comment:
Currently the member agents are only creating events, but decided to let the member agent update, delete as well